### PR TITLE
Add tests to clientutil

### PR DESCRIPTION
**Pull request by cviecco**
Coverage at 64.3%... enought for pull request. Still need to write end-to-end tests

### CODE REVIEW

File: cmd/getcreds/main.go [hunk 1]
```diff
@@ -108,88 +105,75 @@ func loadVerifyConfigFile(configFilename string) (AppConfigFile, error) {
...
-	config, err := loadVerifyConfigFile(*configFilename)
+	fileWriter, err := bodyWriter.CreateFormFile("pubkeyfile", pubKeyFilename)
 	if err != nil {
-		panic(err)
+		return nil, err
 	}
-
-	usr, err := user.Current()
+	// open file handle
+	fh, err := os.Open(pubKeyFilename)
```
Comment (rgooch) [inline cmd/getcreds/main.go]: Please add: defer fh.Close()

File: cmd/getcreds/main.go [hunk 2]
```diff
@@ -225,9 +265,9 @@ func main() {
 	//TODO: change deletion for atomic rename
 	os.Remove(certPath)
 	err = ioutil.WriteFile(certPath, cert, 0644)
-	if !success {
-		log.Fatal("failed to get creds")
-	}
+	//if !success {
+	//	log.Fatal("failed to get creds")
+	//}
```
Comment (rgooch) [inline cmd/getcreds/main.go]: This (and other commented-out code) will be cleaned up, right?

File: cmd/getcreds/main.go (hunk 1 above)
Comment (cviecco) [inline cmd/getcreds/main.go]: Changes Applied in next patch

**Comment by cviecco** [comment]
Another place to put comments?

**Pull request merged**